pkg/u8g2/Makefile.dep

```diff
@@ -0,0 +1,3 @@
+USEMODULE += xtimer
+FEATURES_REQUIRED += periph_gpio
+CFLAGS += -DU8X8_WITH_USER_PTR
```

Comment:
I don't see this flag being used.

Reply:
It is used by u8g2 to define the function u8g2_SetUserPtr() which is used in my proposed adaptation.

Reply:
Can you add a new line below?